src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <Shipping>true</Shipping>
     <IncludeInVSIX>true</IncludeInVSIX>
+    <PackProject>true</PackProject>
```

Comment:
nit: why are the changes in 2 different places :D compare this to nuget.packagemanager.ui.csproj

Reply:
any project that has to be packed has to explicitly set this. it defaults to false.

Reply:
nah, my question/nit was the formatting :)
Half the projects had the new projects one under the other line, the others didn't :)